RAT/utils/plotting.py

```diff
@@ -191,3 +194,33 @@ def plot_ref_sld(data: PlotEventData, fig: Figure = None, delay: bool = True):
         plt.pause(0.005)
     
     return fig
+
+
+def plot_ref_sld(problem, results, block: bool = False):
+    """
+    Plots the reflectivity and SLD profiles.
+
+    Parameters
+    ----------
+    problem : Project
+              An instance of the Project class
```

Comment:
As written, this should be an instance of "ProblemDefinition", not the project class.